update and clarify what has to be done where (#310)
voxpupuli/voxpupuli.github.io: _docs/releasing_version.md

---

Creating a release is a two step process:

1. Prepare the release — setup everything so that peer review can happen and when everything is ready…
2. Do the actual release

## Preparing a release

> This has to be done on a __*private*__ fork.

Reviewer: maybe personal instead of private? Because GitHub also has the option to make a repository private/public, this sentence could confuse people. And it's not technical required to do it on a fork, but we recommend it:
```suggestion
This should be done on a __*personal*__ fork.
```

Author: yes its not techn. needed, but everybody does it like this and most people are not allowed to push branches to our repos? are they?
